shared_fields_common.view.lkml

```diff
@@ -114,6 +114,15 @@ view: shared_fields_common {
     group_label: "Location"
   }
 
+  dimension: city_and_region {
```

Comment:
`geo_city_and_region` would match all other dimensions in the Location section.

Reply:
Changed it, though I think down the road we likely want to remove the "geo_" from all of them.

Reply:
We could also use the label parameter on these to change how they appear in the UI.